oak-blob-cloud/src/main/java/org/apache/jackrabbit/oak/blob/cloud/s3/Utils.java

```diff
@@ -482,6 +574,17 @@ private static void configureBuilder(final S3BaseClientBuilder builder, final Pr
 
         builder.endpointOverride(getEndPointUri(prop, accReq, region));
         builder.crossRegionAccessEnabled(Boolean.parseBoolean(prop.getProperty(S3Constants.S3_CROSS_REGION_ACCESS)));
+
+        // Disable checksums (replaces deprecated checksumValidationEnabled)
+        builder.requestChecksumCalculation(RequestChecksumCalculation.WHEN_REQUIRED);
+        builder.responseChecksumValidation(ResponseChecksumValidation.WHEN_REQUIRED);
+
```

Comment:
Again defaults changed?

Reply:
Good catch, these needs to be disabled only for GCP.